scripts/checkpatch_inc.sh

```diff
@@ -3,11 +3,12 @@
 CHECKPATCH="${CHECKPATCH:-checkpatch.pl}"
 # checkpatch.pl will ignore the following paths
 CHECKPATCH_IGNORE=$(echo \
+		core/arch/arm/include/arm{32,64}.h \
+		core/arch/arm/kernel/elf_load_private.h \
```

Comment:
So `core/arch/arm/kernel/elf_load_private.h` is so hideous that we'll need to ignore it?

Reply:
Not really ;-) `checkpatch.pl` complains about the function pointer declaration in `struct elf_load_state` and about the `DO_ACTION()` macro that contains return statements.
If you prefer we can just ignore those warnings.